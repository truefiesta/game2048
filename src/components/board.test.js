import React from "react";
import renderer from "react-test-renderer";
import Board from "./board";

describe(`Board snapshot`, () => {
    it(`renders`, () => {
        const board = renderer.create(
            <Board/>
        )
        .toJSON();

        expect(board).toMatchSnapshot();
    });
});