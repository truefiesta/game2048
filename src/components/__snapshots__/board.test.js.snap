// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board snapshot renders 1`] = `
<div
  className="board"
/>
`;
